packages/kit/src/views/ReferFriends/pages/YourReferred/index.tsx

```
@@ -107,29 +116,53 @@
+                <SizableText size="$bodyMd" color="$textSubdued">
+                  {intl.formatMessage(
+                    {
+                      id: ETranslations.referral_your_referred_wallets_more_address,
+                    },
+                    {
+                      amount: item.total > 999 ? '999+' : item.total,
+                    },
+                  )}
+                </SizableText>
+              </ListItem>
+              <Divider mx="$5" />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)